packages/near-membrane-base/src/environment.ts

```diff
@@ -105,32 +104,6 @@ export class VirtualEnvironment implements MembraneBroker {
         // placeholder since this will be assigned in construction
     }
 
-    // eslint-disable-next-line class-methods-use-this
-    getBlueRef(
```

Comment:
removing abstraction since the env will not be a thing that we will be able to share with both sides of the fence due to the callable boundary.